aip/general/0008.md

```diff
@@ -7,22 +7,40 @@ placement:
   order: 40
 ---
 
-# AIP Style guide
+# AIP Style and Guidance
 
 AIP stands for **API Improvement Proposal**, which is a design document
-providing high-level, concise documentation for API development. The goal is for
-these documents to serve as the source of truth for API-related documentation at
-Google and the way API teams discuss and come to consensus on API guidance.
+providing high-level, concise documentation for API design and development. The
+goal is for these documents to serve as the source of truth for API-related
+documentation at Google and the way API teams discuss and come to consensus on
```

Comment:
I see this is pre-existing, but is this part of the AIPs specific to Google?

Reply:
yes. The general AIPs actually say something a bit different: https://aip-dev.github.io/aip.dev/1.